### PR TITLE
doc: add private packages usage documentation

### PR DESCRIPTION
**Pull request by Obero**

### CODE REVIEW

**Comment by Obero** [comment]
So you don't have to generate it locally :

image

File: content/reference/admin/private_locations/private_packages/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Private Packages"
+description: "How to install and use a private repository on your Control Plane"
+lead: "Keep your simulations packages privately in your infrastructure, and use them along private locations"
```
Comment (stb13579) [inline content/reference/admin/private_locations/private_packages/index.md:4]:
```suggestion
lead: "Store your simulations' packages privately in your infrastructure, and use them with private locations"
```

File: content/reference/admin/private_locations/private_packages/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+date: 2023-08-09T12:00:00+00:00
+lastmod: 2023-08-09T12:00:00+00:00
+weight: 22033
+---
+
+# Introduction
+
+Using private locations allows you to run your simulations in your private cloud. Now, what about your simulations packages ? They may hold private information, so you may want to store them in your private cloud also.
```
Comment (stb13579) [inline content/reference/admin/private_locations/private_packages/index.md:12]:
```suggestion
Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
```

File: content/reference/admin/private_locations/private_packages/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Your control plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
```
Comment (stb13579) [inline content/reference/admin/private_locations/private_packages/index.md:14]:
```suggestion
Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
```

File: content/reference/admin/private_locations/private_packages/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+# Private packages
+
+## Infrastructure
+
+{{< img src="private_packages_general_architecture.png" alt="Infrastructure schema" >}}
+
+{{< alert info >}}
+Private packages are based on private locations. You can not use them with managed locations.
```
Comment (stb13579) [inline content/reference/admin/private_locations/private_packages/index.md:23]:
```suggestion
Private Packages are based on Private Locations. You can not use them with managed locations.
```

File: content/reference/admin/private_locations/private_packages/index.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+{{< alert info >}}
+Before going further, ensure that your AWS S3 bucket is ready to hold your packages.
+{{< /alert >}}
+
+{{< alert warning >}}
+Control plane with private repository needs AWS permissions `s3:PutObject`` and `s3:DeleteObject` on the bucket.
+{{< /alert >}}
+
+Once it is done, just add private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}})  file :
```
Comment (stb13579) [inline content/reference/admin/private_locations/private_packages/index.md:38]:
```suggestion
Once it is done, add the private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}})  file :
```

**Review by stb13579 — APPROVED** [review]
I made a few small suggestions to improve the English. Otherwise thanks for writing the documentation and awesome work on Private Packages!

File: content/reference/admin/private_locations/private_packages/index.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+{{< /alert >}}
+
+{{< alert warning >}}
+Control plane with private repository needs AWS permissions `s3:PutObject`` and `s3:DeleteObject` on the bucket.
+{{< /alert >}}
+
+Once it is done, add the private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}})  file :
+```bash
+control-plane {
+  # ... existing configuration
```
[the remark this answers is not in the document]
Comment (slandelle) [reply content/reference/admin/private_locations/private_packages/index.md:41]: Yes. remove this line.

**Pull request merged**